### PR TITLE
Bug/addressed feedback

### PR DESCRIPTION
**Pull request by angelnikolov**
Restricted the file picker to the supported audio formats only
Ref. https://github.com/Fliplet/fliplet-widget-audio-player/pull/9

### CODE REVIEW

**Comment by tonytlwu** [comment]
If we want to make sure only MP3 files are presented in the File Picker when called by the Audio Player component, we should request the File Picker for only MP3 files -- not by restricting File Picker to only give you MP3 files when asking for all audio formats.

**Comment by angelnikolov** [comment]
@tonytlwu Can you review again? Thanks!

File: js/interface.js [hunk 1]
```diff
@@ -1046,7 +1059,7 @@ function handleUploadingWrongFile() {
 }
 
 function showWrongFileError() {
-  var supportedExtensions = extensionDictionary[data.type].join(', ').toUpperCase();
+  var supportedExtensions = data.fileExtension && data.fileExtension.length ? data.fileExtension.join(', ').toUpperCase() : extensionDictionary[data.type].join(', ').toUpperCase();
```
Comment (tonytlwu) [inline js/interface.js]: Separate the lines.

```suggestion
  var supportedExtensions = data.fileExtension && data.fileExtension.length
    ? data.fileExtension.join(', ').toUpperCase()
    : extensionDictionary[data.type].join(', ').toUpperCase();
```

Comment (angelnikolov) [reply js/interface.js]: @tonytlwu Done!

**Pull request merged**